### ISSUE TITLE
Updating minDate and maxDate doesn't work due to container check

### ISSUE THREAD

**Issue by cjzness** [voronianski/react-pikaday-component#3]
Changes to the minDate and maxDate props are not reflected in the date picker, because the container check prevents the Pikaday object from being recreated:

```
    componentDidUpdate(prevProps) {
        // update if container is set
        if (!prevProps.container && this.props.container) {
```

**Comment by voronianski** [comment]
@cjzness could you provide some example jsfiddle etc.?

**Comment by voronianski** [comment]
@cjzness you can use this link to provide your example - http://esnextb.in/?gist=b7bb48ddc187dc7eece468154361b4a7

**Comment by cjzness** [comment]
Oh that's cool. I have created an example for you: http://esnextb.in/?gist=199122a58318c3a158809ae04681db8f

1. The initial min and max dates (5th and 20th of April) are shown correctly in the date picker.
2. Click "Change Dates".
3. Changed dates are not reflected in the date picker.

Console logging lines are included so that you can see the props are being passed correctly to the component.

**Comment by gliesche** [comment]
Any updates on this?

**Comment by voronianski** [comment]
@cjzness @gliesche fixed in `2.1.1` - https://esnextb.in/?gist=3cce657da1adcaf0f2c89d44647f86cd&exec=yes
